nixos/modules/config/pulseaudio.nix

```diff
@@ -118,6 +118,13 @@ in {
           '';
         };
       };
+
+      displayName = mkOption {
```

Comment:
not that I'm aware of. Also, in practice is it even possible to run multiple X servers with different DISPLAY values on NixOS? I've done it before on other distros, but only by manually running `startx`, and that doesn't work on NixOS.

So, I would also love a generic solution to this, but I think it's a problem for another day and outside of the scope of this diff.